lib/Dialect/SV/SVDialect.cpp

```diff
@@ -98,6 +98,14 @@ StringRef circt::sv::resolveKeywordConflict(StringRef origName,
   }
 }
 
+static bool isValidVerilogCharacterFirst(char ch) {
+  return isalpha(ch) || ch == '_';
+}
+
+static bool isValidVerilogCharacter(char ch) {
+  return isValidVerilogCharacterFirst(ch) || isdigit(ch);
```

Comment:
Supernit, but please use `isDigit` and friends in `llvm/ADT/StringExtras.h` instead of ctypes.h.  `isdigit` in various libc's include a bunch of (rather dubious) unicode nonsense that we don't want.

Reply:
Excellent point. Fixed :+1: